src/main/java/org/mule/extension/runtime/Expirable.java

```diff
@@ -0,0 +1,20 @@
+package org.mule.extension.runtime;
+
+/**
+ * An entity which can or cannot be considered elegible for expiration
+ * according to a {@link ExpirationPolicy}
+ *
+ * @since 1.0
+ */
+public interface Expirable
+{
+
+    /**
+     * Determins if {@code this} instance should be expired based on the given
+     * {@code expirationPolicy}
+     *
+     * @param expirationPolicy a {@link ExpirationPolicy}
+     * @return {@code true} if {@code this} can be expired. {@code false} otherwise
```

Comment:
no... one thing is to be elegible for expiration and a different one is to actually be expired. For example, the config has an expiration of 1 Minute, but the platform only checks for expired configs every 5 minutes... So this method returns true if invoked at second 61, but that doesn't mean that it's actually expired already since performing the expiration is a separate concern